cap_anndata/cap_anndata.py

```diff
@@ -104,10 +112,13 @@ def _read_df(self, h5_group: h5py.Group, columns: List[str]) -> CapAnnDataDF:
         return df
 
     @staticmethod
-    def _read_attr(obj: Union[h5py.Group, h5py.Dataset], attr_name: str) -> any:
+    def _read_attr(obj: Union[h5py.Group, h5py.Dataset], attr_name: str, raise_exception: bool = True) -> any:
```

Comment:
The issue must be fixed upon writing to save compability with AnnData. Please don't modify this method

Reply:
done